tests/test_help.py

```diff
@@ -0,0 +1,15 @@
+# -*- encoding: utf-8
+
+import sys
+
+import pytest
+
+from requests.help import info
+
+
+@pytest.mark.skipif(sys.version_info[:2] != (2,6), reason="Only run on Python 2.6")
+def test_system_ssl_py26():
+    """OPENSSL_VERSION_NUMBER isn't provided in Python 2.6, verify we don't
+    blow up in this case.
+    """
+    assert info()['system_ssl'] == {'version': ''}
```

Comment:
This test file would also pass if there was never anything interesting in `system_ssl`.

How about a second test that asserts that on Pythons >2.6, the `version` key is non-empty?